iiab

```diff
@@ -608,8 +610,13 @@ echo -e "global community, learning from other initiatives in your region and be
 #echo -e "(over Wi-Fi anyway) until you later run 'iiab-hotspot-off'.  CAVEAT: these"
 #echo -e "two commands only work with Raspberry Pi as of 2019-09-30.\n"
 
-echo -ne "HIT [Enter] TO REBOOT, CONFIRMING YOU'VE" '"photographed" THE ABOVE PARAGRAPHS: '
-read ans < /dev/tty
+if ! $($MFABT); then
```

Comment:
Testing dogfooding the same workflow as actual users keeps everyone on the same wavelength (as much as possible, i.e. forcing them to type CTRL-c is a good thing!)

So can we change this line to something like?

```suggestion
if ! $($MFABT) && ! $(ischroot -t); then
```

Reply:
Let me try that in the chroot